oteapi/strategies/parse/image.py

```diff
@@ -48,29 +65,66 @@ class ImageParserResourceConfig(ResourceConfig):
     )
 
 
-class SupportedFormat(Enum):
+class SupportedFormats(Enum):
     """Supported formats for `ImageDataParseStrategy`."""
 
-    JPEG = "JPEG"
-    JPEG2000 = "JPEG2000"
-    PNG = "PNG"
-    GIF = "GIF"
-    TIFF = "TIFF"
-    EPS = "EPS"
+    jpeg = "JPEG"
+    jpg = "JPEG"
+    jp2 = "JPEG2000"
+    png = "PNG"
+    gif = "GIF"
+    tiff = "TIFF"
+    tif = "TIFF"
+    eps = "EPS"
 
 
 class SessionUpdateImageParse(SessionUpdate):
-    """Configuration model for ImageParse."""
+    """Configuration model for ImageParse.
 
-    content: bytes = Field(..., description="Parsed output from ImageParse.")
-    format: SupportedFormat = Field(..., description="The format of the parsed image.")
-    cropped: bool = Field(..., description="Whether or not the image has been cropped.")
+    For
+      - image_mode
+      - image_palette
+      - image_info
+
+    see [Pillow handbook](https://pillow.readthedocs.io/en/stable/handbook/concepts.html) for more details.
+    """
+
+    image_key: str = Field(
+        ...,
+        description="Key with which the image content is stored in datacache.",
+    )
+    image_size: Tuple[int, int] = Field(
+        ...,
+        description="Image size (width, height).",
+    )
+    image_mode: str = Field(
+        ...,
+        description="Image mode. Examples: 'L', 'P', 'RGB', 'RGBA'...",
+    )
```

Comment:
Yes, this can be inferred from the incoming image. 

But this is a SessionUpdate that is specific for the strategy, so it will only be instantiated in this strategy (or am I wrong?). Since an image always will have a size and mode, I thought it would be a good idea to make that explicit by making these fields non-optional.

Reply:
Well, here it's the configuration you're defining. I.e., this will potentially be the user-supplied information. I don't think it makes sense to require the mode and more here.

Reply:
Ah no - you're right. This is the "output", sorry.